### PR TITLE
Big cleanup of measures

### PR DESCRIPTION
**Pull request by nitarshan**
1. We were including the (reparameterized) batch norm weights in calculations before. (Note that this is not _wrong_ relative to previous work, but I think this is more correct).
2. We were not squaring (approximate) spectral norm values before.
3. We were taking the square root of the path norm before - we do so again in the post-processing, so this was being done twice.

The measures have been refactored and rearranged for clarity.

Measure comparison report: https://app.wandb.ai/nitarshan/rgm/reports/Untitled-Report--VmlldzoxOTg3ODY

### CODE REVIEW

File: source/measures.py [hunk 1]
```diff
@@ -103,150 +93,156 @@ def _pacbayes_sigma(
...
-  if use_reparam:
-    reparam(model)
-    reparam(init_model)
+  model = _reparam(model)
+  init_model = _reparam(init_model)
 
   device = next(model.parameters()).device
-  weights_only = get_weights_only(model)
-  init_weights_only = get_weights_only(init_model)
+  num_params = sum(p.numel() for p in model.parameters() if p.requires_grad)
```
Comment (ethancaballero) [inline source/measures.py]: Should't this exclude params in batchnorm layer?

**Review by ethancaballero — CHANGES_REQUESTED** [review]
